pages/instructorAssessmentAccess/instructorAssessmentAccess.ejs

```diff
@@ -7,21 +7,23 @@
     <script>
       $(function () {
           $('[data-toggle="popover"]').popover({sanitize: false})
+          $('[data-toggle="tooltip"]').tooltip()
       });
     </script>
     <%- include('../partials/navbar'); %>
     <div id="content" class="container-fluid">
       <%- include('../partials/assessmentSyncErrorsAndWarnings'); %>
 
-      <div class="card mb-4">
-        <div class="card-header bg-primary text-white d-flex align-items-center">
-          <%= assessment_set.name %> <%= assessment.number %>: Access
-        </div>
+      <details open class="card mb-4">
+        <summary class="card-header bg-primary text-white d-flex align-items-center">
+          <%= assessment_set.name %> <%= assessment.number %>: Defined Access rules
+        </summary>
```

Comment:
This would be a brand-new pattern for PL. Without any visual indications that this can be collapsed, I'm a bit hesitant to thumbs-up this. I'd recommend adding something like the down/left chevron that GitHub uses on the "File changed" page of a PR:

<img width="198" alt="Screen Shot 2022-04-07 at 15 04 10" src="https://user-images.githubusercontent.com/1476544/162327340-20832610-643a-459c-872b-d858a317f20e.png">

<img width="230" alt="Screen Shot 2022-04-07 at 15 05 07" src="https://user-images.githubusercontent.com/1476544/162327369-5047597e-ab1e-4a53-ae87-2378c30276cb.png">

Reply:
I think the Bootstrap theme already has some styles for collapsible sections too. The grading results divs on question pages use them.

Reply:
This was something I've originally added when I first created the PR in 2020, but I don't feel strongly about. I'm happy to replace it with regular divs instead. That said, removing `d-flex` from the summary element causes the browser's default chevron to show up:

<img width="223" alt="image" src="https://user-images.githubusercontent.com/1004907/162532527-712dd8ce-78a7-4bb6-a61e-37d4d882ce8d.png">

Any preference?